src/js/claims-status/utils/appeal-helpers.jsx

```diff
@@ -4,35 +4,35 @@ import moment from 'moment';
 export const hearingDescriptions = {
   video: {
     title: 'You have asked for a videoconference hearing.',
-    description: <p>You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
+    description: <p className="next-action-text">You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
   },
   travel_board: { // eslint-disable-line camelcase
     title: 'You have asked for a travel board hearing. ',
-    description: <p>You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
+    description: <p className="next-action-text">You will get a letter in the mail at least 30 days before your hearing is scheduled letting you know the date, time, and location of the hearing.</p>
   },
   central_office: { // eslint-disable-line camelcase
     title: 'You asked for a Board hearing in Washington, DC',
-    description: <p>You’ll get a letter in the mail at least 30 days before your hearing is scheduled. It will let you know the date and time of the hearing. The Board’s central office hearings take place at 425 Eye Street, NW, Washington, DC, 20001.</p>
+    description: <p className="next-action-text">You’ll get a letter in the mail at least 30 days before your hearing is scheduled. It will let you know the date and time of the hearing. The Board’s central office hearings take place at 425 Eye Street, NW, Washington, DC, 20001.</p>
   },
 };
 
-export function appealStatusDescriptions(lastEvent, previousHistory = []) {
+export function appealStatusDescriptions(lastEvent, previousHistory = [], style) {
```

Comment:
I don't think having this style parameter is necessary. The classes passed into it seem to always try to make the first `p` element in a group have a top margin of 0. Can we create a class that sets the top margin to 0 directly, and add it to the first `p` element that needs it in each description? Will that cause any problems you can think of?